### PR TITLE
Victor/primary

### PR DESCRIPTION
**Pull request by Victor-chiemerie**
# Highlights 🥇 

## What I have done
✅ Setup component linking using `<NavLink/>`

## Why
🔴 Check if any error with the setup

### Changes details
Please add @Victor-chiemerie in your comment so that I get notified
## Missed anything?
- [ ] Explained the purpose of this PR.
- [ ] Self reviewed the PR.
- [ ] Added or updated test cases.
- [ ] Informed of breaking changes, testing and migrations (if applicable).
- [ ] Updated documentation (if applicable).
- [ ] Attached screenshots (if applicable).

## Followed Rules: 
- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
-  Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-  Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
-  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
-  Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
+import './Navbar.scss';
+import { NavLink } from 'react-router-dom';
 import logo from '../../assets/ai.png';
 
+const links = [
+  { path: '/', text: 'Rockets' },
+  { path: 'missions', text: 'Missions' },
+  { path: 'profile', text: 'My Profile' },
+];
+
```
Comment (roniy68) [inline src/components/Navbar/Navbar.jsx:11]: 👍 I like the idea to create an object of routes. 😄

**Review by roniy68 — APPROVED** [review]
# 🔥 Status :  Approved ✅ 
---

## Great  Job @Victor-chiemerie 



There's nothing wrong with the implementation. 
### Highlights 🎖️ 
---
## Followed Rules: 
- Made sure [no linter errors](https://github.com/microverseinc/linters-config) 🔴 
-  Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) ✅ 
-  Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) ✅ 
-  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). ✅ 
-  Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). ✅ 

### Code Readability :mag_right:
- [ ] Code is easy to read and understand. :white_check_mark:
- [ ] Code is commented where necessary. :pencil:
- [ ] Code is organized and structured. :file_folder:

### Functionality :computer:
- [ ] Code performs its intended function.  ✔️ 
- [ ] Code handles errors and edge cases appropriately. ✔️ 
- [ ] Code is optimized for performance where necessary. ✔️ 

### Testing :microscope:
- [ ] Code has been tested thoroughly. :test_tube:
- [ ] Test cases cover all possible scenarios. :heavy_check_mark:
- [ ] Test cases are well documented and easy to understand. :books:

### Security :lock:
- [ ] Code is secure and follows best practices. :shield:
- [ ] Code does not introduce vulnerabilities. :no_entry_sign:

### Documentation :page_facing_up:
- [ ] Code is well documented. :memo:
- [ ] Documentation is easy to understand. :thumbsup:
- [ ] Documentation is up to date. :calendar:

### Conclusion :tada:
- [ ] Code review has been completed. :white_check_mark:
- [ ] Any issues have been reported and discussed. :loudspeaker:
- [ ] Code is ready for merge into main branch. :rocket:


***NOTE: Please mention @roniy68 in your comments so that I can get Notification. Happy Coding :-)***

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Victor-chiemerie) [reply src/components/Navbar/Navbar.jsx:11]: Thanks man 😄
